extractor/src/main/scala/com/digitalasset/extractor/writers/postgresql/MultiTableDataFormat.scala

```diff
@@ -63,15 +64,18 @@ class MultiTableDataFormat(
       // base case when there's nothing to merge into
 
       val takenNames = if (schemaPerPackage) {
-        state.templateToTable
+        state.templateToTable.view
           .filterKeys(_.packageId == id.packageId)
+          .toMap
           .values
+          .view
           .map(_.withOutSchema)
```

Comment:
```suggestion
          .map(_._2.withOutSchema)
```
Would this be equivalent?

Reply:
very good point 7e80ce20ea13d3b371f6d923d13ba8f94c5719e9. On 2.13, the view has `.values` but not on 2.12 and I didn’t realize how stupid it had become once I added the `toMap` in between.